desktop/assets/texts/tr-TR/preferences_dialog.ftl

```diff
@@ -28,3 +28,7 @@ gamemode = GameMode
 gamemode-tooltip = GameMode, bilgisayarınıza ve/veya Ruffle'a geçici olarak bir dizi optimizasyon uygular. Ruffle, yalnızca bir film oynatıldığında GameMode'u talep eder.
 gamemode-default = Varsayılan
 gamemode-default-tooltip = GameMode yalnızca güç tercihi yüksek olarak ayarlandığında etkinleştirilecektir.
+# See for context https://wiki.archlinux.org/title/Input_method
+ime-enabled = Girdi yöntemi
+ime-enabled-experimental = (deneyimsel)
```

Comment:
This seems to mean something along the lines of "experiential", whatever that is...?

Reply:
(probably a typo somewhere)

Reply:
pinged the contributor on CrowdIn, and picked the "100% match (global)" suggestion from there in the meantime